traits/api.py

```diff
@@ -23,7 +23,7 @@
 
 from __future__ import absolute_import
 
-from .trait_base import Uninitialized, Undefined, Missing, Self, python_version
+from .trait_base import Uninitialized, Undefined, Missing, Self
```

Comment:
Urk! I didn't realise we were even exposing `python_version` in `traits.api`.

I just did a quick check over the rest of ETS and I don't find any use of this export.

FWIW, this is a user-visible change (though I sincerely hope that no user cares), so we should remember to note it in the changelog when the time comes.

Reply:
Updating the changelog now instead of later.